subprojects/ff-nvcodec-headers.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-git]
```

Comment:
Git wraps are not permitted in wrapdb, only tarballs.

Reply:
ff-nvcodec-headers does not release tarballs, and I do not expect them to ever do that. That's why I added it and pointed it to a tag.